### PR TITLE
Hello

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by Mshivam2409** [comment]
@crimsoncandy A set is prone to user-made errors i.e. " Nikhil" <  "Akshit" .

**Comment by ghost** [comment]
@Mshivam2409 Just trim the names then.

**Comment by Mshivam2409** [comment]
Why are we sorting multiple times ?

**Comment by ghost** [comment]
@Mshivam2409 Well I see only one sort, unless there is some other function calling sort.

**Comment by Mshivam2409** [comment]
Inserting n items in set is equivalent to inserting n items in array and sorting. Unless you know red black trees, you can assume set calls sort() within.

**Review by Mshivam2409 — CHANGES_REQUESTED** [review]
I still do not see any improvement in `update_authors.py`.

File: .contrib/update_authors.py [hunk 1]
```diff
@@ -4,16 +4,16 @@
 
 cargo = toml.load('Cargo.toml')
 
-authors = sorted(os.listdir('./.contrib'))
+authors = os.listdir('./.contrib')
 
-authors_list = []
+authors_list = set()
```
Comment (Mshivam2409) [inline .contrib/update_authors.py]: Set is already a sorted structure.

File: .contrib/update_authors.py [hunk 2]
```diff
@@ -4,16 +4,16 @@
...
 
 for author in authors:
     if author.endswith("json"):
         js = json.load(open("./.contrib/"+author))
-        authors_list.append("{} <{}>".format(js["name"],js["email"]))
+        authors_list.add("{} <{}>".format(js["name"].strip(), js["email"].strip()))
 
-cargo["package"]["authors"] = authors_list
+cargo["package"]["authors"] = sorted(authors_list)
```
Comment (Mshivam2409) [inline .contrib/update_authors.py]: We are sorting a set here again ?

**Review by Mshivam2409 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**